cardano_node_tests/tests/reqs_conway.py

```diff
@@ -120,6 +120,13 @@ def __r(id: str) -> requirements.Req:
 cip078 = __r("CIP078")
 cip079 = __r("CIP079")
 
+cli085 = __r("CLI085")
```

Comment:
This requirements should be acording the User Stories.
https://tests.cardano.intersectmbo.org/test_results/requirements/chang_user_stories_system_tests.html